grammars/perl.cson

```diff
@@ -637,7 +637,7 @@
     ]
   }
   {
-    'begin': '(?<=\\(|\\{|~|&)\\s*((\\/))'
+    'begin': '(?<=\\(|\\{|~|&|\\||if|unless|^)\\s*((\\/))'
```

Comment:
This is obviously something really small and definitely doesn't need to block this PR, but the curly bracket `{` doesn't need to be escaped.  Also, I *think* `^` is usually placed at the beginning of the match.